### ISSUE TITLE
[question] How to get the TD with MQTT Subscribe

### ISSUE THREAD

**Issue by egekorkan** [eclipse/thingweb.node-wot#144]
For HTTP and CoAP, the TD is always found at `http://myIP:port/myDevice`. Where is this for the MQTT? I can think of the servient publishing its TD upon successful connection to the broker with retain set to true. A python example with the paho library from Eclipse would be:
```
def on_connect(client, userdata, flags, rc):
    client.publish("td", td, retain=True)
```

**Comment by danielpeintner** [comment]
@egekorkan let's do the same as for http and coap, report td based thing `title`

**Comment by egekorkan** [comment]
PR available at #162

**Comment by egekorkan** [comment]
Available on master branch, closing
